app/views/extractions/new.html.erb

```diff
@@ -1,5 +1,132 @@
-<h1>New Extraction</h1>
+<%= simple_form_for(@extraction) do |f| %>
+<div class= "extrac-new-main-container">
+  <div class="actions-buttons">
+    <%= link_to 'Back', 'javascript:history.back()', class: 'btn btn-outline-secondary', onclick: "if (document.referrer === '') { window.location.href = '#{pages_dashboard_path}'; }" %>
+    <%= f.button :submit, class: 'btn btn-primary' %>
+  </div>
+  <h1>New Extraction</h1>
 
-<%= render 'form', extraction: @extraction %>
+  <% @default_params ||= {} %>
 
-<%= link_to 'Back', extractions_path %>
+    <div class="extrac-new-step">
+      <div class="extrac-new-acc-container">
+        <div class="extrac-new-acc-single">
+          Coffee
+          <div class="extrac-new-acc-tile">
+          </div>
+          <%= f.input :user_coffee_id,
+              collection: current_user.user_coffees, as: :select,
+              include_blank: true,
+              label: false,
+              value_method: :id,
+              selected: @default_params[:user_coffee_id],
+              input_html: { class: 'form-control', multiple: false } %>
+        </div>
+          <div>
+            Grinder
+            <div class="extrac-new-acc-tile">
+            </div>
+            <%= f.input :grinder_id,
+                collection: current_user.accessories.grinder, as: :select,
+                include_blank: true,
+                label: false,
+                value_method: :id,
+                selected: @default_params[:grinder_id],
+                input_html: { class: 'form-control', multiple: false } %>
+                <%= f.input :grinder_set, label: "Grinder Position", input_html: { class: 'form-control', value: @default_params[:grinder_set] } %>
+          </div>
+        <div>
+          Brewer
+          <div class="extrac-new-acc-tile">
+          </div>
+          <%= f.input :brewing_method_id,
+                collection: current_user.accessories.brewing_method.pluck(:name, :id),
+                label: false,
+                selected: @default_params[:brewing_method_id],
+                input_html: { class: 'form-control' } %>
+          <%= f.input :water_temperature, label: "Water Temperature", input_html: { class: 'form-control', value: @default_params[:water_temperature] } %>
+        </div>
+      </div>
+
+      <div class="extrac-new-acc-secondary">
+        <h4>Optional Accessories</h4>
+        <%= f.input :accessory_ids,
+                    collection: current_user.accessories.where.not(accessory_type: Accessory::BREWING_METHODS + ['Grinder']),
```

Comment:
Maybe create a scope in the accessory model to access miscellaneous accessories

Reply:
Je vais peut etre même creer d autre scopes pour d autre besoins UX specifiques